features/edition-attachments.feature

```diff
@@ -4,7 +4,7 @@ Feature: Managing attachments on editions
   I want to attach files and additional HTML content to publications and consultations
   In order to support the publications and consultations with statistics and other relevant documents
 
-  @design-system-wip
+  @design-system-only
```

Comment:
Looks good. Just one question. Are we going to make sure we do this forward once functionality is ported so the tests are quicker since they're essentially duplicates at that point.

Do we need to be a bit of an audit of previous releases or have you covered that here? It looks like you caught a couple at least from previous ones.